diff --git a/cmake/TargetOther.cmake b/cmake/TargetOther.cmake
index 53c91542..35553fbc 100644
--- a/cmake/TargetOther.cmake
+++ b/cmake/TargetOther.cmake
@@ -6,7 +6,9 @@ if(CMAKE_SOURCE_DIR STREQUAL PROJECT_SOURCE_DIR)
     target_link_libraries(lslver PRIVATE lsl)
     set_target_properties(lslver PROPERTIES FOLDER "liblsl")
     if(LSL_INSTALL AND NOT IOS)
-        install(TARGETS lslver RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
+        install(TARGETS lslver
+            BUNDLE DESTINATION "${CMAKE_INSTALL_BINDIR}"
+            RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
     endif()
 
     if(NOT WIN32 AND LSL_TOOLS)
