diff --git a/cmake/Installation.cmake b/cmake/Installation.cmake
index dcb803b8..9404a64e 100644
--- a/cmake/Installation.cmake
+++ b/cmake/Installation.cmake
@@ -55,6 +55,11 @@ install(TARGETS ${LSLTargets}
     ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
     FRAMEWORK DESTINATION ${CMAKE_INSTALL_FRAMEWORK_DIR}
 )
+if(MSVC AND NOT LSL_BUILD_STATIC)
+    install(FILES $<TARGET_PDB_FILE:lsl>
+        DESTINATION "${CMAKE_INSTALL_BINDIR}" OPTIONAL
+    )
+endif()
 # Unfortunately, `INCLUDES DESTINATION` does not work.
 # PUBLIC_HEADER does not work because it flattens the tree.
 # FILE_SET is preferable but does not work with frameworks.
diff --git a/cmake/TargetLib.cmake b/cmake/TargetLib.cmake
index e26e4852..3cf63dba 100644
--- a/cmake/TargetLib.cmake
+++ b/cmake/TargetLib.cmake
@@ -30,6 +30,11 @@ endif()
 #  but are used by the internal tests.
 target_link_libraries(lsl PRIVATE lslobj)
 
+if(MSVC AND NOT LSL_BUILD_STATIC)
+    target_compile_options(lsl PRIVATE $<$<CONFIG:Release>:/Zi>)
+    target_link_options(lsl PRIVATE "$<$<CONFIG:Release>:/DEBUG;/OPT:REF;/OPT:ICF>")
+endif()
+
 # Set the include directories for the lsl target.
 # Note: We had to link lslobj as a PRIVATE dependency, therefore we must manually expose the include directories
 if(APPLE AND LSL_FRAMEWORK)
