From 4cc86ec5e3c3c8d33dd05c60cee56db757e73a5c Mon Sep 17 00:00:00 2001
From: myd7349 <myd7349@gmail.com>
Date: Thu, 24 Apr 2025 22:54:21 +0800
Subject: [PATCH] Fix #233

---
 CMakeLists.txt | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f86958f7..8260d2fc 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -119,8 +119,9 @@ add_library(lslobj OBJECT
 if(LSL_BUNDLED_PUGIXML)
 	message(STATUS "Using bundled pugixml")
 	target_sources(lslobj PRIVATE thirdparty/pugixml/pugixml.cpp)
-	target_include_directories(lslobj SYSTEM PUBLIC
-		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/pugixml>)
+	target_include_directories(lslobj
+		PUBLIC
+			$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/thirdparty/pugixml>)
 else()
 	message(STATUS "Using system pugixml")
 	find_package(pugixml REQUIRED)
@@ -169,8 +170,9 @@ find_package(Threads REQUIRED)
 add_library(lslboost INTERFACE)
 if(LSL_BUNDLED_BOOST)
 	message(STATUS "Using bundled Boost")
-	target_include_directories(lslboost SYSTEM INTERFACE
-		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lslboost>)
+	target_include_directories(lslboost
+		INTERFACE
+			$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/lslboost>)
 else()
 	message(STATUS "Using system Boost")
 	find_package(Boost REQUIRED)
@@ -194,7 +196,7 @@ target_include_directories(lslobj
 		$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/src>
 	)
 target_include_directories(lslobj
-	SYSTEM PUBLIC
+	PUBLIC
 		$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/thirdparty/loguru>
 		$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/thirdparty/asio>
 )
-- 
2.47.0.windows.2

